### DISCUSSION TITLE
Multilingual documents and interlinear text

### DISCUSSION THREAD

**Discussion by gjvnq** [knowledgefutures/pubpub#2294] [Ideas]
Hi, first I would like to say that pubpub is almost my dream CMS!

Secondly, I did some poking around in the sandbox and I would like to ask if there plans to add any of the following features.

- Linking pub translations in a "standard area/fields" (like each language version is its own separate pub)
- Pubs in multiple languages (like as single thing rather than separate one so the a multilingual pub has unified comments, reviews, etc.)
- Parallel text (like showing an original text and a translation side by side) (like what the [paracol](https://linorg.usp.br/CTAN/macros/latex/contrib/paracol/paracol-man.pdf) LaTeX package does)
- [Interlinear text/gloss](https://en.wikipedia.org/wiki/Interlinear_gloss) (kinda like `<ruby>` but on steroids)

**Comment by kyledenlinger** [comment]
You may want to check out the [2022 Global Deep-Sea Capacity Assessment community](https://deepseacapacity.oceandiscoveryleague.org/) to see how they handled translations. I think they use the connections feature to relate pubs to their translations, and the collections feature to group pubs by language.

**Reply by kaylendwyer** [reply]
Hi, resurrecting this question...

I like how the Global Deep-Sea community handles the translations and I think that might be a workable solution for the project I'm working on, but parallel texts translations would be amazing. The project involves open annotation of English translated Greek texts. Only the English text is annotated, not the Greek, but it would be great if the Greek could be there in parallel as an option. (So maybe on the left side as a two-column layout) 

Are there ways I can accomplish a two column layout already?
